styles/layers.h

```diff
@@ -73,12 +80,21 @@ class Compose<BLACK, L1> {
   }
 };
 
-
 template<class BASE, class ... LAYERS> struct LayerSelector {};
 template<class BASE> struct LayerSelector<BASE> { typedef BASE type; };
+
+// Drop fully transparent first item (Alpha 0) when there are exactly 2 args.
 template<class BASE, class L1> struct LayerSelector<AlphaL<BASE, Int<0>>, L1> { typedef L1 type; };
+
+// Drop fully transparent first item (Alpha 0) when there are 3+ args.
```

Comment:
I think the correct fix for this is to change line 82 from:

```
  typedef typename LayerSelector<Compose<BASE, L1>, REST...>::type type;
```

to

```
  typedef typename LayerSelector<LayerSelector<BASE, L1>::type, REST...>::type type;
```

I think that's what I originally intended anyways.
That way everything gets broken down into two-layer calls to LayerSelector, and we don't have to special-case three-layer stuff.

Reply:
oops. actually deleted this time.